flow/project.py

```diff
@@ -3065,10 +3066,18 @@ def dotted_get(mapping, key):
 
                 status["parameters"] = {}
                 for parameter in parameters:
-                    status["parameters"][parameter] = shorten(
-                        str(self._alias(dotted_get(statepoint, parameter))),
-                        param_max_width,
-                    )
+                    if not parameter.startswith("doc."):
+                        status["parameters"][parameter] = shorten(
+                            str(self._alias(dotted_get(statepoint, parameter))),
+                            param_max_width,
+                        )
+                    else:
+                        if document is None:
+                            document = self.open_job(id=aggregate_id).document()
```

Comment:
Reusing the `job` from above should be faster than opening the job again.
```suggestion
                            document = job.document()
```

Reply:
@kidrahahjo To your question above, the document is not cached by default because it is not "fixed" like the state point. The state point for a Job is meant to be treated as immutable, since signac's storage is based on the hash of that state point. However, we can't guarantee that a document is unchanged between reads. For the scope of rendering status parameters, it is perfectly fine to assume that the document is fixed and cache it to avoid re-reading and parsing the file for every parameter requested.